src/QRCodeInterface.ts

```diff
@@ -30,12 +30,6 @@ import {
  */
 interface QRCodeInterface {
 
-    /**
-     * The base64 representation of the QR Code content.
-     * @var {string}
-     */
-    base64?: string|undefined;
```

Comment:
you can remove the `|undefined` part but why remove the full property description?

Reply:
it's never actually used in the code